### PR TITLE
Cloudedit

### PR DESCRIPTION
**Pull request by aheifetz**
Fixed typo's in require statements.   May need to do a clean install.  Also cloud be differences in the way Mac's and ubuntu (my test server on Cloud9.io) handle case sensitivity.

### CODE REVIEW

**Comment by aheifetz** [comment]
I also added a "taggable" sub class for resources per Chri's suggestion.  Not sure I like the inhertence model 100% but it seems to work.  Easy tagging becomes important for ops.

File: lib/Taggable.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+};
+require('util').inherits(Taggable, Resource);
+
+
+Taggable.registerPropertyPrototypes = function(klass, propertyMap){
+
+    return Resource.registerPropertyPrototypes(klass, propertyMap);
+    
+}
```
Comment (owap) [inline lib/Taggable.js]: Do we need to include this function? Does `registerPropertyPrototypes` pass onto the Taggable class on line 28 when we inherit its functions?

Comment (aheifetz) [inline lib/Taggable.js]: its didn't - I think because its not on the prototype.  you're pushing on node knowledge.

Comment (aheifetz) [inline lib/Taggable.js]: I mean pushing my node knowledge :)

**Pull request merged**